apps/webpack.config.js

```diff
@@ -1 +1,114 @@
-module.exports = require('./webpack').config;
+var _ = require('lodash');
```

Comment:
I'm going to undo all the changes in this file. So please disregard. Was just using it temporarily to generate webpack profiling information that could be uploaded to their analyzer.